data/.~lock.bank_data.csv#

```diff
@@ -0,0 +1 @@
+,iqra,iqra-HP-Pavilion-Laptopp-15-cu1xxx,06.06.2022 16:59,file:///home/iqra/.config/libreoffice/4;
```

Comment:
First of all, we did not upload the data files to the repository. Add these files(data/.~lock.bank_data.csv#) in the git ignore file.